Order.Management/Order.cs

```diff
@@ -1,20 +1,17 @@
-﻿using System;
+﻿using Order.Management.ToyBlocks;
 using System.Collections.Generic;
-using System.Text;
 
 namespace Order.Management
 {
-    abstract class Order
+    public class Order
     {
         public string CustomerName { get; set; }
         public string Address { get; set; }
-        public string DueDate { get; set; }
+        public string DueDate { get; set; } // TODO: should probably be DateTime
         public int OrderNumber { get; set; }
-        public List<Shape> OrderedBlocks { get; set; }
+        public List<Shape> OrderedBlocks { get; set; } = new List<Shape>();
```

Comment:
`OrderedBlocks` data structure should be its own class with properties for each Shape type i.e.
```
public class OrderedBlocks
{
    public List<Square> Squares { get; set; }
    public List<Triangle> Triangles { get; set; }
    public List<Circle> Circles { get; set; }
}
```